scripts/node/root/weave-daemonset.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: extensions/v1beta1
```

Comment:
Where does this file "weave-daemonset.yaml" come from? Could you add an URL in a comment?